### PR TITLE
Extra credit part 4

### PR DESCRIPTION
**Pull request by mkotyak**

### CODE REVIEW

File: Set/Card/CardViewBuilder.swift [hunk 1]
```diff
@@ -2,8 +2,8 @@ import Foundation
 import SwiftUI
 
 class CardViewBuilder {
-    func build(for card: CardModel) -> CardView {
-        let viewModel = CardViewModel(card: card)
+    func build(for card: CardModel, _ isColorBlindModeEnabled: Bool) -> CardView {
+        let viewModel = CardViewModel(card: card, isColorBlindModeEnabled)
```
Comment (garbalinskiy) [inline Set/Card/CardViewBuilder.swift]: Don't use underscore for second attribute

File: Set/Card/CardViewModel.swift [hunk 2]
```diff
@@ -7,9 +7,11 @@ class CardViewModel: ObservableObject {
     }
     
     @Published private var card: CardModel
+    var isColorBlindModeEnabled: Bool
```
Comment (garbalinskiy) [inline Set/Card/CardViewModel.swift]: isColorBlindModeEnabled is constant and `let` should be used

File: Set/Game/GameViewModel.swift [hunk 3]
```diff
@@ -12,6 +12,7 @@ class GameViewModel: ObservableObject {
     private var countdown = Constants.playerTurnDuration
     private var timer: Timer?
     @Published var timerTitle = ""
+    var isColorBlindModeEnabled: Bool
```
Comment (garbalinskiy) [inline Set/Game/GameViewModel.swift]: `let`

**Review by garbalinskiy — APPROVED** [review]
Very good

File: Set/Card/CardViewBuilder.swift (hunk 1 above)
Comment (mkotyak) [reply Set/Card/CardViewBuilder.swift]: fixed

File: Set/Card/CardViewModel.swift (hunk 2 above)
Comment (mkotyak) [reply Set/Card/CardViewModel.swift]: fixed

File: Set/Game/GameViewModel.swift (hunk 3 above)
Comment (mkotyak) [reply Set/Game/GameViewModel.swift]: fixed

**Pull request merged**